web-client/src/app/views/pay/pay.module.ts

```diff
@@ -17,6 +18,6 @@ import { PayPage } from './pay.page';
     SharedModule,
     InputMaskModule.forRoot({ inputSelector: 'input', isAsync: true }),
   ],
-  declarations: [PayPage],
+  declarations: [PayPage, PayFromToComponent],
```

Comment:
It might make more sense to have a shared module under `src/app/components` to declare the components, rather than having the components under `src/app/components` but their declarations under `src/app/views`?

Reply:
💯